### ISSUE TITLE
Turbiniactl status

### ISSUE THREAD

**Issue by beamcodeup** [google/turbinia#172]
Currently, `turbiniactl status` invokes cloud functions that live on the zone specified in the config file:
https://github.com/google/turbinia/blob/master/turbiniactl#L440

Based on my observation in my GCP, deploy_gcf.py pushed the function definition files to GCF on US-CENTRAL1 from my GCS instance on US-WEST1. However, when running turbiniactl status, it made an attempt of invoking the functions on US-WEST1. Obviously, this failed because they don't exist on that region.

```
$ ./turbiniactl status
[DEBUG] Setting up PSQ Task Manager requirements on project testtest-turbinia
[DEBUG] Connecting to PubSub Subscription on turbinia-dev
[DEBUG] Calling Cloud Function [gettasks] with args [{u'instance': u'turbinia-dev', u'kind': u'TurbiniaTask'}]
Traceback (most recent call last):
  File "./turbiniactl", line 455, in <module>
    request_id=args.request_id, all_fields=args.all_fields)
  File "./turbiniactl", line 149, in PrintTaskStatus
    request_id)
  File "./turbiniactl", line 112, in GetTaskData
    response = function.ExecuteFunction('gettasks', func_args)
  File "/home/turbinia/turbinia/turbinia/lib/google_cloud.py", line 231, in ExecuteFunction
    function_name, e))
turbinia.TurbiniaException: Error calling cloud function [gettasks]: <HttpError 400 when requesting https://cloudfunctions.googleapis.com/v1beta2/projects/testtest-turbinia/locations/us-west1/functions/gettasks:call?alt=json returned "The request has errors">
```
The [documentation](https://cloud.google.com/sdk/gcloud/reference/beta/functions/deploy) does not explain how it decides which region to use if --region is not explictly used.

So, the proposed solution to create a GCF region reference in the config file and have the CTL use that information in order to invoke functions in the correct region by using --region.

**Comment by beamcodeup** [comment]
As of Oct 2017, Cloud Functions are only available on US-CENTRAL1 per https://stackoverflow.com/questions/46902502/use-region-other-than-us-central1-for-google-cloud-functions. For now, we'll hardcode GCF_REGION = 'US-CENTRAL1' in the config.

**Comment by aarontp** [comment]
Fixed in #173
